queries.sql

```diff
@@ -13,4 +13,45 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
 
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
+BEGIN; 
+update animals set species='unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species='digimon' WHERE name like '%mon';
+
+UPDATE animals SET species='pokemon' WHERE species='';
+
+COMMIT;
+
+BEGIN; DELETE from animals;
+
+ROLLBACK;
+
+ BEGIN; DELETE from animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT sp_1;
+
+UPDATE animals SET weight_kg=weight_kg*-1;
+
+ROLLBACK TO sp_1;
+
+
+ UPDATE animals SET weight_kg=weight_kg*-1 where weight_kg<0;
+
+ SELECT count(*) FROM animals;
+
+SELECT count(*) FROM animals WHERE escape_attempts=0;
+
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals WHERE neutered=true OR neutered=false;
+SELECT MAX(weight_kg),MIN(weight_kg) FROM animals WHERE species='digimon';
```

Comment:
- [ ] Great job so far however you did not write queries to answer the following questions:

- What is the minimum and maximum weight of each type of animal?
eg 
```sql
  SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
   GROUP BY species;
```
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 eg :
```sql
   SELECT species, AVG(escape_attempts) FROM animals
   GROUP BY species
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/31/12';
```

- Kindly make this and add a screenshots of their queries results to the pull request description 📸